ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/wizards/product/BaseProductCreationOperation.java

```diff
@@ -67,11 +66,8 @@ public BaseProductCreationOperation(IFile file) {
 
 	@Override
 	protected void execute(IProgressMonitor monitor) throws CoreException, InvocationTargetException, InterruptedException {
-		monitor.beginTask(PDEUIMessages.BaseProductCreationOperation_taskName, 2);
 		createContent();
```

Comment:
I think the monitor should actually be splitted and passed down to createcontent and open files ...